content/blog/20250514_fluentd-v1.16.9-has-been-released.md

```diff
@@ -0,0 +1,28 @@
+# Fluentd v1.16.9 has been released
+
+Hi users!
+
+We have released v1.16.9 on 2025-05-14.
+ChangeLog is [here](https://github.com/fluent/fluentd/blob/v1.16/CHANGELOG.md#release-v1169---20250514).
+
+This release is a maintenance release of v1.16 series.
+In this release, we have fixed a critical bug of the previous version, v1.16.8.
+Please do not use v1.16.8.
+
+This release will be bundled for upcoming `fluent-package` LTS version v5.0.7!
+
+## Bug Fixes
+
+### Windows: Fix bug where the service accidentally stops after starting
+
+In the previous version v1.16.8, the Windows service accidentally stops after starting without stopping the supervisor and worker processes.
```

Comment:
```suggestion
In the previous version v1.16.8, the Windows service (fluentdwinsvc) accidentally stops after starting without stopping the supervisor and worker processes.
```

Reply:
It might be better to describe "what" service actually.

Reply:
Thanks! Applied!